### PR TITLE
Make skeleton code from DB to HTTP response cycle.

### LINKED ISSUE #13: Initialize Transactor on startup

### PR DESCRIPTION
**Pull request by skyhills13**
close #13 #15 #17 

## Refactor
- Change bean like classes from class to object
- Simplify higher kinded type

## Feature
- Make dummy api for getting a value/values from Database.

### CODE REVIEW

**Comment by skyhills13** [comment]
Done making DB configs configurable and reformatting some lines.
(I mix issues only this time.)

**Comment by skyhills13** [comment]
@Kevin-Lee Done updating PR!

File: src/main/scala/io/skyhills/jaguarscala/MainServer.scala [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def helloWorldService: HttpService[IO] = TestService.service
+
+    def transactionService: HttpService[IO] = TransactionService.service
+
+    def wishService: HttpService[IO] = WishService.service
+
+
+    def stream(implicit ec: ExecutionContext): fs2.Stream[IO, StreamApp.ExitCode] =
+        BlazeBuilder[IO]
+            .bindHttp(8080, "0.0.0.0")
```
Comment (Kevin-Lee) [inline src/main/scala/io/skyhills/jaguarscala/MainServer.scala]: Probably for next PR, you'd better consider having the port number and address from the config file.

Comment (skyhills13) [reply src/main/scala/io/skyhills/jaguarscala/MainServer.scala]: Okay!

**Pull request merged**